# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/thebes/gfxPlatformMac.cpp
# Commit: 10b6bafbbd9a
# Full Hash: 10b6bafbbd9ae3ea5539d2524959d491be9d30a9
# Author: Jonathan Kew <jkew@mozilla.com>
# Date: 2022-08-17 09:10:29
# Regressor Bug: 1740530
# File Overlap Count: 1
# Description:
#   Bug 1740530 - patch 7 - Route COLRv1 fonts on macOS through our COLR-rendering path; only COLRv0 can use the native Core Text support. r=gfx-reviewers,lsalzman
#   
#   Depends on D153880
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D152162
# ==============================================================================

diff -r 1c9205a0193f -r 10b6bafbbd9a gfx/thebes/gfxPlatformMac.cpp
--- a/gfx/thebes/gfxPlatformMac.cpp	Tue Aug 16 12:48:33 2022 +0000
+++ b/gfx/thebes/gfxPlatformMac.cpp	Tue Aug 16 12:48:34 2022 +0000
@@ -142,10 +142,6 @@
   mFontAntiAliasingThreshold = ReadAntiAliasingThreshold();
 
   InitBackendPrefs(GetBackendPrefs());
-
-  if (nsCocoaFeatures::OnHighSierraOrLater()) {
-    mHasNativeColrFontSupport = true;
-  }
 }
 
 gfxPlatformMac::~gfxPlatformMac() { gfxCoreTextShaper::Shutdown(); }
